### PR TITLE
Show pending projects in the report

### PR DESCRIPTION
**Pull request by aanil**

### CODE REVIEW

File: daily_read/daily_report.py [hunk 1]
```diff
@@ -23,11 +23,12 @@
...
 STATUS_DESCRIPTIONS = {
-    "All Raw data Delivered": "The data has been made available through NGIs delivery system",
-    "All Samples Sequenced": "Sequencing (including potential resequencing) of all samples has been finished",
+    "All Raw data Delivered": "The data has been made available through NGIs delivery system.",
+    "All Samples Sequenced": "Sequencing (including potential resequencing) of all samples has been finished.",
     "Library QC finished": "Library QC is a quality control of the sequencing library produced either by NGI or supplied by you, depending on the type of project.",
     "Reception Control finished": "Reception Control consists of NGI staff measuring e.g. concentration and volume for the samples received.",
-    "Samples Received": "The samples have been received and registered at NGI",
+    "Samples Received": "The samples have been received and registered at NGI.",
+    "Pending": "The order has been set up but the samples have not yet been received by NGI.",
```
Comment (alneberg) [inline daily_read/daily_report.py]:
```suggestion
    "Pending": "The order has been set up but the samples have not yet been received or registered by NGI.",
```

Comment (alneberg) [reply daily_read/daily_report.py]: Not sure if I should use "and" or "or" in my suggestion though. You probably know better.

**Review by alneberg — APPROVED** [review]
Wow, I expected this to be much more difficult! 👍

File: daily_read/daily_report.py (hunk 1 above)
Comment (aanil) [reply daily_read/daily_report.py]: I guess 'or' is better? They won't show up until we enter it in LIMS.

**Pull request merged**